code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -24,6 +24,8 @@
 
 /obj/effect/clockwork/sigil/attack_hand(mob/user)
 	if(iscarbon(user) && !user.stat)
+		if(is_servant_of_ratvar(user) == user.a_intent != "harm")
```

Comment:
USE THE DEFINE also there is no reason you ever want to remove sigils on reebe so make this station-only or something